### PR TITLE
Server status: add doc type 'uppmax' or 'nas' when saving to Status DB

### PR DESCRIPTION
**Pull request by kate-v-stepanova**

### CODE REVIEW

**Comment by Galithil** [comment]
do we have to keep the ugly printf-like syntax line 27 ?

Nothing else to complain about.

**Comment by kate-v-stepanova** [comment]
sorry, you missed it in the previous pull requests. the only line which was changed is this one: https://github.com/SciLifeLab/TACA/pull/168/files#diff-6baad2dbb0fc9f7874749012eedfc3caR125

**Comment by Galithil** [comment]
yeah i've seen that, it's just that I don't like finding %s in the code again.

File: taca/server_status/server_status.py [hunk 1]
```diff
@@ -9,185 +9,186 @@
...
+        logging.error(e.message)
+        raise
+
+    db = couch['server_status']
+    logging.info('Connection established')
+    for key in data.keys(): # data is dict of dicts
+        server = data[key] # data[key] is dictionary (the command output)
+        server['name'] = key # key is nas url or uppmax project
+        server['time'] = datetime.datetime.now().isoformat() # datetime.datetime(2015, 11, 18, 9, 54, 33, 473189) is not JSON serializable
+        server['server_type'] = 'uppmax' if 'uppmax' in server['name'] else 'nas'
```
Comment (vezzi) [inline taca/server_status/server_status.py]: seems fine I have an hard time putting together the three pull request but as long as it tested it is fine

**Pull request merged**